src/overtone/helpers/atom.clj

```diff
@@ -0,0 +1,53 @@
+(ns
+ ^{:doc "Atom helper fns"}
+ overtone.helpers.atom)
+
+(defn atom-view
```

Comment:
I get why you have this, but I guess (in the future), it would be better if we could intercept a key lookup (by using something like potemkin, for example) in `Inst` directly, but needs a discussion for sure.

Reply:
Having sat with this for a while, I'm going to drop the `atom-view` and simply remove `:volume` and `:pan` atoms from Inst. My expectation is that no Overtone user will notice.